src/util.js

```diff
@@ -32,7 +32,20 @@ function loadYamlFile(filePath) {
 }
 
 
+function loadFile(filePath) {
+  let fileData;
+  if (fs.existsSync(`${filePath}.js`)) {
+    fileData = require(`${filePath}.js`) // eslint-disable-line import/no-dynamic-require, global-require
+  } else if (fs.existsSync(`${filePath}.yaml`)) {
+    fileData = loadYamlFile(`${filePath}.yaml`)
+  } else if (fs.existsSync(filePath)) {
+    fileData = loadYamlFile(filePath)
```

Comment:
Slight duplication here. Just do if (.yaml or no extension) - loadYaml.

Reply:
I got a good point, I have one ideia on how to make this code better. will push in a few minute